Source/WebCore/page/scrolling/mac/ScrollerMac.mm

```diff
@@ -428,7 +443,7 @@ - (void)invalidate
 
 void ScrollerMac::updateMinimumKnobLength(int minimumKnobLength)
 {
-    if (m_minimumKnobLength == minimumKnobLength)
+    if (m_minimumKnobLength == minimumKnobLength && minimumKnobLength)
```

Comment:
Why the check for `minimumKnobLength` here? Explain in the commit message.

Reply:
Removed.